src/main/java/com/codejoust/main/model/User.java

```diff
@@ -37,8 +45,23 @@
      */
     private String sessionId;
 
+    // This column holds the primary key of the account
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "account_table_id")
+    private Account account;
+
     // This column holds the primary key of the room (not the roomId variable)
     @ManyToOne(fetch = FetchType.EAGER)
     @JoinColumn(name = "room_table_id")
     private Room room;
+
+    // This column holds the primary key of the game report
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "game_report_table_id")
+    private GameReport gameReport;
```

Comment:
Does each user only have up to one GameReport? I thought multiple game reports could belong to each room and user session

Reply:
You're right, I made a mistake here - I've removed this field and updated the `List<User>` field within the `GameReport` class.